CONTRIBUTING.md

```diff
@@ -135,7 +135,10 @@ To do so, run
 pip install -e .'[all]'
 ```
 from within your local `tedana` repository. This should ensure all packages are correctly organized and linked on your user profile.
-
+If the above command didnot worked for you try using the following command :
+```
+pip install -e .[all]
+```
```

Comment:
Rather than adding more steps to try, I think it would be best to make our standard installation instructions work for as many people as possible. Why not change the instructions above to `pip install -e .[all]`?

Reply:
That does not work on every shell unfortunately; in my zsh (MacOS default) it attempts to expand the shell and then there are no valid expansions, so it exits with error:
```
zsh: no matches found: .[all]
```
so we do indeed need quotes for UNIX systems and non-quotes for Windows.

Reply:
That's unfortunate, but I'm glad to know definitively that that's the case.

Reply:
But it does omit dev dependencies. I pushed some changes to simplify the language a bit, confirmed working on the relevant platforms.